client/cmd/up.go

```diff
@@ -135,6 +147,25 @@ func runInDaemonMode(ctx context.Context, cmd *cobra.Command) error {
 		return err
 	}
 
+  if cmd.Flag(disableAutoConnectFlag).Changed {
```

Comment:
@oskardotglobal for the daemon mode, you need to move the flag check to after the loginRequest initiallization:

https://github.com/netbirdio/netbird/pull/1161/files#diff-0db67eedcdec93665cc9dea08a99e38d7ed16f94db2e98f842af921340815542R206

And it should be similar to the rosenpassEnabled flag. 

This will require an update of the grpc protocol file with a new message field. See https://github.com/netbirdio/netbird/blob/main/client/proto/daemon.proto#L59

Reply:
I updated the protocol but I can't seem to find what to do with the value

Reply:
Where do I actually update the config after receiving the field inside the message?

Reply:
Never mind, figured it out